nixd/lspserver/include/lspserver/Connection.h

```diff
@@ -40,9 +40,15 @@ class InboundPort {
 
   JSONStreamStyle StreamStyle = JSONStreamStyle::Standard;
 
-  bool readStandardMessage(std::string &JSONString);
+  /// Read one message as specified in the LSP standard.
+  /// A Language Server Protocol message starts with a set of
+  /// HTTP headers, delimited  by \r\n, and terminated by an empty line (\r\n).
+  std::optional<llvm::Expected<llvm::json::Value>>
```

Comment:
Don't like this double-wrapped json value. I just made some local changes to address this, can you [allow me to edit the PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)?

Reply:
turned it on :)